sdk/instrumentation/instrumentation-opentelemetry/test/internal/works.spec.ts

```diff
@@ -0,0 +1,68 @@
+import chai, { assert } from "chai";
+import { chaiAzureTrace } from "./chai";
+chai.use(chaiAzureTrace);
+import { InMemoryInstrumenter } from "./inMemoryInstrumenter";
+import { createTracingClient, TracingClient, useInstrumenter } from "@azure/core-tracing";
+// https://github.com/maorleger/azure-sdk-for-js/compare/instrumentation-opentelemetry...maorleger:spike-instrumenter-tests?expand=1
+
+// So like we can have some low-level support with ways to inspect the instrumenter
+// check the spans, etc.
+describe.only("in-memory", () => {
+  const instrumenter = new InMemoryInstrumenter();
+
+  beforeEach(() => {
+    useInstrumenter(instrumenter);
+  });
+
+  it("works", async () => {
+    const instrumenter = new InMemoryInstrumenter();
```

Comment:
Maybe instead this can be 

```ts
const client = new FakeClientToTest();
await client.fakeMethod();
assert.equal(instrumenter.startedSpans.length, 1)
```
...
```